### ISSUE TITLE
[BUG] df.loc[:, [column_name]] returns Series and not DataFrame

### ISSUE THREAD

**Issue by JulianWgs** [rapidsai/cudf#4207]
**Describe the bug**
When using `.loc` and a list with only one item as the column name cudf returns a Series, but pandas a DataFrame which also more consistent. When using multiple columns cudf returns a DataFrame.

**Steps/Code to reproduce bug**
```python
import pandas as pd
import cudf

print(type(pd.DataFrame({"A": [1, 2], "B": [3, 4]}).loc[:, ["A"]]))
print(type(cudf.DataFrame({"A": [1, 2], "B": [3, 4]}).loc[:, ["A"]]))
```

Output:
```
<class 'pandas.core.frame.DataFrame'>
<class 'cudf.core.series.Series'>
```

**Expected behavior**
Code should return a DataFrame in both cases.

**Environment overview (please complete the following information)**
- Google Colab with [installation script](https://raw.githubusercontent.com/rapidsai/notebooks-contrib/branch-0.12/utils/rapids-colab.sh) (Updated script)
- Choose 0.12
- Install `!conda install -y -c anaconda cffi`

**Environment details**
<details><summary>Click here to see environment details</summary><pre>
     
     **git***
     Not inside a git repository
     
     ***OS Information***
     DISTRIB_ID=Ubuntu
     DISTRIB_RELEASE=18.04
     DISTRIB_CODENAME=bionic
     DISTRIB_DESCRIPTION="Ubuntu 18.04.3 LTS"
     NAME="Ubuntu"
     VERSION="18.04.3 LTS (Bionic Beaver)"
     ID=ubuntu
     ID_LIKE=debian
     PRETTY_NAME="Ubuntu 18.04.3 LTS"
     VERSION_ID="18.04"
     HOME_URL="https://www.ubuntu.com/"
     SUPPORT_URL="https://help.ubuntu.com/"
     BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
     PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
     VERSION_CODENAME=bionic
     UBUNTU_CODENAME=bionic
     Linux 6ea82b97f267 4.14.137+ #1 SMP Thu Aug 8 02:47:02 PDT 2019 x86_64 x86_64 x86_64 GNU/Linux
     
     ***GPU Information***
     Thu Feb 20 07:39:34 2020
     +-----------------------------------------------------------------------------+
     | NVIDIA-SMI 440.48.02    Driver Version: 418.67       CUDA Version: 10.1     |
     |-------------------------------+----------------------+----------------------+
     | GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
     | Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
     |===============================+======================+======================|
     |   0  Tesla P100-PCIE...  Off  | 00000000:00:04.0 Off |                    0 |
     | N/A   36C    P0    32W / 250W |    431MiB / 16280MiB |      0%      Default |
     +-------------------------------+----------------------+----------------------+
     
     +-----------------------------------------------------------------------------+
     | Processes:                                                       GPU Memory |
     |  GPU       PID   Type   Process name                             Usage      |
     |=============================================================================|
     +-----------------------------------------------------------------------------+
     
     ***CPU***
     Architecture:        x86_64
     CPU op-mode(s):      32-bit, 64-bit
     Byte Order:          Little Endian
     CPU(s):              2
     On-line CPU(s) list: 0,1
     Thread(s) per core:  2
     Core(s) per socket:  1
     Socket(s):           1
     NUMA node(s):        1
     Vendor ID:           GenuineIntel
     CPU family:          6
     Model:               79
     Model name:          Intel(R) Xeon(R) CPU @ 2.20GHz
     Stepping:            0
     CPU MHz:             2200.000
     BogoMIPS:            4400.00
     Hypervisor vendor:   KVM
     Virtualization type: full
     L1d cache:           32K
     L1i cache:           32K
     L2 cache:            256K
     L3 cache:            56320K
     NUMA node0 CPU(s):   0,1
     Flags:               fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss ht syscall nx pdpe1gb rdtscp lm constant_tsc rep_good nopl xtopology nonstop_tsc cpuid tsc_known_freq pni pclmulqdq ssse3 fma cx16 pcid sse4_1 sse4_2 x2apic movbe popcnt aes xsave avx f16c rdrand hypervisor lahf_lm abm 3dnowprefetch invpcid_single ssbd ibrs ibpb stibp fsgsbase tsc_adjust bmi1 hle avx2 smep bmi2 erms invpcid rtm rdseed adx smap xsaveopt arat md_clear arch_capabilities
     
     ***CMake***
     /usr/local/bin/cmake
     cmake version 3.12.0
     
     CMake suite maintained and supported by Kitware (kitware.com/cmake).
     
     ***g++***
     /usr/bin/g++
     g++ (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0
     Copyright (C) 2017 Free Software Foundation, Inc.
     This is free software; see the source for copying conditions.  There is NO
     warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
     
     
     ***nvcc***
     /usr/local/cuda/bin/nvcc
     nvcc: NVIDIA (R) Cuda compiler driver
     Copyright (c) 2005-2018 NVIDIA Corporation
     Built on Sat_Aug_25_21:08:01_CDT_2018
     Cuda compilation tools, release 10.0, V10.0.130
     
     ***Python***
     /usr/local/bin/python
     Python 3.6.7
     
     ***Environment Variables***
     PATH                            : /usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/tools/node/bin:/tools/google-cloud-sdk/bin:/opt/bin
     LD_LIBRARY_PATH                 : /usr/lib64-nvidia
     NUMBAPRO_NVVM                   :
     NUMBAPRO_LIBDEVICE              :
     CONDA_PREFIX                    :
     PYTHON_PATH                     :
     
     ***conda packages***
     /usr/local/bin/conda
     # packages in environment at /usr/local:
     #
     # Name                    Version                   Build  Channel
     _libgcc_mutex             0.1                 conda_forge    conda-forge
     _openmp_mutex             4.5                       0_gnu    conda-forge
     arrow-cpp                 0.15.0           py36h090bef1_2    conda-forge
     asn1crypto                0.24.0                   py36_0
     blinker                   1.4                        py_1    conda-forge
     bokeh                     1.4.0                    py36_0    conda-forge
     boost-cpp                 1.70.0               h8e57a91_2    conda-forge
     brotli                    1.0.7             he1b5a44_1000    conda-forge
     bzip2                     1.0.8                h516909a_2    conda-forge
     c-ares                    1.15.0            h516909a_1001    conda-forge
     ca-certificates           2020.1.1                      0    anaconda
     cachetools                3.1.1                      py_0    conda-forge
     cairo                     1.16.0            hfb77d84_1002    conda-forge
     certifi                   2019.11.28               py36_0    anaconda
     cffi                      1.14.0           py36h2e261b9_0    anaconda
     cfitsio                   3.470                hb60a0a2_2    conda-forge
     chardet                   3.0.4                 py36_1003    conda-forge
     click                     7.0                        py_0    conda-forge
     cloudpickle               1.3.0                      py_0    conda-forge
     conda                     4.5.4                    py36_0
     conda-env                 2.6.0                h36134e3_1
     cryptography              2.8              py36h72c5cf5_1    conda-forge
     cudatoolkit               10.0.130                      0    nvidia
     cudf                      0.11.0                   py36_0    rapidsai
     cudnn                     7.6.0                cuda10.0_0    nvidia
     cugraph                   0.11.0                   py36_0    rapidsai
     cuml                      0.11.0          cuda10.0_py36_0    rapidsai
     cupy                      6.7.0            py36he57b8b9_1    conda-forge
     curl                      7.68.0               hf8cf82a_0    conda-forge
     cuspatial                 0.11.0                   py36_0    rapidsai
     cython                    0.29.15          py36he1b5a44_0    conda-forge
     cytoolz                   0.10.1           py36h516909a_0    conda-forge
     dask                      2.11.0                     py_0    conda-forge
     dask-core                 2.11.0                     py_0    conda-forge
     dask-cudf                 0.11.0                   py36_0    rapidsai
     decorator                 4.4.1                      py_0    conda-forge
     distributed               2.11.0                   py36_0    conda-forge
     dlpack                    0.2                  he1b5a44_1    conda-forge
     double-conversion         3.1.5                he1b5a44_2    conda-forge
     expat                     2.2.9                he1b5a44_2    conda-forge
     fastavro                  0.22.9           py36h516909a_0    conda-forge
     fastrlock                 0.4             py36he1b5a44_1000    conda-forge
     fontconfig                2.13.1            h86ecdb6_1001    conda-forge
     freetype                  2.10.0               he983fc9_1    conda-forge
     freexl                    1.0.5             h14c3975_1002    conda-forge
     fsspec                    0.6.2                      py_0    conda-forge
     gcsfs                     0.6.0                      py_0    conda-forge
     gdal                      2.4.4            py36h5f563d9_0    conda-forge
     geos                      3.8.0                he1b5a44_0    conda-forge
     geotiff                   1.5.1                h38872f0_8    conda-forge
     gettext                   0.19.8.1          hc5be6a0_1002    conda-forge
     gflags                    2.2.2             he1b5a44_1002    conda-forge
     giflib                    5.1.7                h516909a_1    conda-forge
     glib                      2.58.3          py36h6f030ca_1002    conda-forge
     glog                      0.4.0                he1b5a44_1    conda-forge
     google-auth               1.11.2                     py_0    conda-forge
     google-auth-oauthlib      0.4.1                      py_2    conda-forge
     grpc-cpp                  1.23.0               h18db393_0    conda-forge
     hdf4                      4.2.13            hf30be14_1003    conda-forge
     hdf5                      1.10.5          nompi_h3c11f04_1104    conda-forge
     heapdict                  1.0.1                      py_0    conda-forge
     icu                       64.2                 he1b5a44_1    conda-forge
     idna                      2.9                        py_1    conda-forge
     jinja2                    2.11.1                     py_0    conda-forge
     joblib                    0.14.1                     py_0    conda-forge
     jpeg                      9c                h14c3975_1001    conda-forge
     json-c                    0.13.1            h14c3975_1001    conda-forge
     kealib                    1.4.10            h58c409b_1005    conda-forge
     krb5                      1.16.4               h2fd8d38_0    conda-forge
     libblas                   3.8.0               14_openblas    conda-forge
     libcblas                  3.8.0               14_openblas    conda-forge
     libcudf                   0.11.0               cuda10.0_0    rapidsai
     libcugraph                0.11.0               cuda10.0_0    rapidsai
     libcuml                   0.11.0               cuda10.0_0    rapidsai
     libcumlprims              0.11.0               cuda10.0_0    nvidia
     libcurl                   7.68.0               hda55be3_0    conda-forge
     libcuspatial              0.11.0               cuda10.0_0    rapidsai
     libdap4                   3.20.4               hd3bb157_0    conda-forge
     libedit                   3.1.20170329      hf8c457e_1001    conda-forge
     libevent                  2.1.10               h72c5cf5_0    conda-forge
     libffi                    3.2.1             he1b5a44_1006    conda-forge
     libgcc-ng                 9.2.0                h24d8f2e_2    conda-forge
     libgdal                   2.4.4                h2b6fda6_0    conda-forge
     libgfortran-ng            7.3.0                hdf63c60_5    conda-forge
     libgomp                   9.2.0                h24d8f2e_2    conda-forge
     libiconv                  1.15              h516909a_1005    conda-forge
     libkml                    1.3.0             h4fcabce_1010    conda-forge
     liblapack                 3.8.0               14_openblas    conda-forge
     libllvm8                  8.0.1                hc9558a2_0    conda-forge
     libnetcdf                 4.7.3           nompi_h9f9fd6a_101    conda-forge
     libnvstrings              0.11.0               cuda10.0_0    rapidsai
     libopenblas               0.3.7                h5ec1e0e_6    conda-forge
     libpng                    1.6.37               hed695b0_0    conda-forge
     libpq                     12.1                 hd9ab2ff_0    conda-forge
     libprotobuf               3.8.0                h8b12597_0    conda-forge
     librmm                    0.11.0               cuda10.0_0    rapidsai
     libspatialite             4.3.0a            ha48a99a_1034    conda-forge
     libssh2                   1.8.2                h22169c7_2    conda-forge
     libstdcxx-ng              9.2.0                hdf63c60_2    conda-forge
     libtiff                   4.1.0                hfc65ed5_0    conda-forge
     libuuid                   2.32.1            h14c3975_1000    conda-forge
     libxcb                    1.13              h14c3975_1002    conda-forge
     libxgboost                0.90.rapidsdev1      cuda10.0_1    rapidsai/label/xgboost
     libxml2                   2.9.10               hee79883_0    conda-forge
     llvmlite                  0.31.0           py36h8b12597_0    conda-forge
     locket                    0.2.0                      py_2    conda-forge
     lz4-c                     1.8.3             he1b5a44_1001    conda-forge
     markupsafe                1.1.1            py36h516909a_0    conda-forge
     msgpack-python            1.0.0            py36hc9558a2_0    conda-forge
     nccl                      2.4.6.1              cuda10.0_0    nvidia
     ncurses                   6.1               hf484d3e_1002    conda-forge
     numba                     0.48.0           py36hb3f55d8_0    conda-forge
     numpy                     1.18.1           py36h95a1406_0    conda-forge
     nvstrings                 0.11.0                   py36_0    rapidsai
     oauthlib                  3.0.1                      py_0    conda-forge
     olefile                   0.46                       py_0    conda-forge
     openjpeg                  2.3.1                h981e76c_3    conda-forge
     openssl                   1.1.1                h7b6447c_0    anaconda
     packaging                 20.1                       py_0    conda-forge
     pandas                    0.24.2           py36hb3f55d8_1    conda-forge
     parquet-cpp               1.5.1                         2    conda-forge
     partd                     1.1.0                      py_0    conda-forge
     pcre                      8.44                 he1b5a44_0    conda-forge
     pillow                    7.0.0            py36hefe7db6_0    conda-forge
     pip                       20.0.2                     py_2    conda-forge
     pixman                    0.38.0            h516909a_1003    conda-forge
     poppler                   0.67.0               h14e79db_8    conda-forge
     poppler-data              0.4.9                         1    conda-forge
     postgresql                12.1                 hc63931a_0    conda-forge
     proj                      6.3.0                hc80f0dc_0    conda-forge
     psutil                    5.7.0            py36h516909a_0    conda-forge
     pthread-stubs             0.4               h14c3975_1001    conda-forge
     py-xgboost                0.90.rapidsdev1  cuda10.0py36_1    rapidsai/label/xgboost
     pyarrow                   0.15.0           py36h8b68381_1    conda-forge
     pyasn1                    0.4.8                      py_0    conda-forge
     pyasn1-modules            0.2.7                      py_0    conda-forge
     pycosat                   0.6.3            py36h0a5515d_0
     pycparser                 2.19                       py_2    conda-forge
     pyjwt                     1.7.1                      py_0    conda-forge
     pynvml                    8.0.4                      py_0    conda-forge
     pyopenssl                 19.1.0                     py_1    conda-forge
     pyparsing                 2.4.6                      py_0    conda-forge
     pysocks                   1.7.1                    py36_0    conda-forge
     python                    3.6.7             h357f687_1006    conda-forge
     python-dateutil           2.8.1                      py_0    conda-forge
     pytz                      2019.3                     py_0    conda-forge
     pyyaml                    5.3              py36h516909a_0    conda-forge
     re2                       2020.01.01           he1b5a44_0    conda-forge
     readline                  8.0                  hf8c457e_0    conda-forge
     requests                  2.23.0                   py36_0    conda-forge
     requests-oauthlib         1.2.0                      py_0    conda-forge
     rmm                       0.11.0                   py36_0    rapidsai
     rsa                       4.0                        py_0    conda-forge
     ruamel_yaml               0.15.37          py36h14c3975_2
     scikit-learn              0.22.1           py36hcdab131_1    conda-forge
     scipy                     1.4.1            py36h921218d_0    conda-forge
     setuptools                45.2.0                   py36_0    conda-forge
     six                       1.14.0                   py36_0    conda-forge
     snappy                    1.1.8                he1b5a44_1    conda-forge
     sortedcontainers          2.1.0                      py_0    conda-forge
     sqlite                    3.30.1               hcee41ef_0    conda-forge
     tblib                     1.6.0                      py_0    conda-forge
     thrift-cpp                0.12.0            hf3afdfd_1004    conda-forge
     tk                        8.6.10               hed695b0_0    conda-forge
     toolz                     0.10.0                     py_0    conda-forge
     tornado                   6.0.3            py36h516909a_4    conda-forge
     tzcode                    2019a             h516909a_1002    conda-forge
     uriparser                 0.9.3                he1b5a44_1    conda-forge
     urllib3                   1.25.7                   py36_0    conda-forge
     wheel                     0.34.2                     py_1    conda-forge
     xerces-c                  3.2.2             h8412b87_1004    conda-forge
     xgboost                   0.90.rapidsdev1  cuda10.0py36_1    rapidsai/label/xgboost
     xorg-kbproto              1.0.7             h14c3975_1002    conda-forge
     xorg-libice               1.0.10               h516909a_0    conda-forge
     xorg-libsm                1.2.3             h84519dc_1000    conda-forge
     xorg-libx11               1.6.9                h516909a_0    conda-forge
     xorg-libxau               1.0.9                h14c3975_0    conda-forge
     xorg-libxdmcp             1.1.3                h516909a_0    conda-forge
     xorg-libxext              1.3.4                h516909a_0    conda-forge
     xorg-libxrender           0.9.10            h516909a_1002    conda-forge
     xorg-renderproto          0.11.1            h14c3975_1002    conda-forge
     xorg-xextproto            7.3.0             h14c3975_1002    conda-forge
     xorg-xproto               7.0.31            h14c3975_1007    conda-forge
     xz                        5.2.4             h14c3975_1001    conda-forge
     yaml                      0.2.2                h516909a_1    conda-forge
     zict                      1.0.0                      py_0    conda-forge
     zlib                      1.2.11            h516909a_1006    conda-forge
     zstd                      1.4.3                h3b9ef0a_0    conda-forge
     
</pre></details>

**Comment by galipremsagar** [comment]
Fixed by: #4613
